### PR TITLE
Add PowerReviews

### PR DESCRIPTION
**Pull request by 40504**

### CODE REVIEW

File: entries/p/powerreviews.com.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+  "PowerReviews": {
+    "domain": "powerreviews.com",
```
Comment (Carlgo11) [inline entries/p/powerreviews.com.json:3]: Their `https://powerreviews.com` certificate isn't valid.
```suggestion
    "domain": "powerreviews.com",
    "url": "https://www.powerreviews.com"
```

**Review by Carlgo11 — CHANGES_REQUESTED** [review]

File: entries/p/powerreviews.com.json (hunk 1 above)
Comment (Carlgo11) [reply entries/p/powerreviews.com.json:3]: do
```BASH
curl -I https://powerreviews.com/
```
Does it fail?

**Review by phallobst — APPROVED** [review]

File: entries/p/powerreviews.com.json (hunk 1 above)
Comment (40504) [reply entries/p/powerreviews.com.json:3]: You are right, the certificate isn't valid

**Review by Carlgo11 — APPROVED** [review]

**Pull request merged**